iscsi/iscsi.go

```diff
@@ -30,6 +30,10 @@ const IqnGlob = "target/iscsi/iqn*"
 // reading runtime status
 const TargetCore = "target/core"
 
+// DevicePath static path /sys/devices/rbd/[0-9]* for rbd devices to
+// read at runtime status
+const DevicePath = "devices/rbd"
```

Comment:
Are `DevicePath` and `TargetCore` still needed outside this package?  Or can they be changed to `devicePath` and `targetCore`?

Reply:
yup, I update all the static path and pushed.